hash/rx78_cart.xml

```diff
@@ -109,14 +113,16 @@ The monitor will expect you to load a program from cassette.
 		</part>
 	</software>
 
-	<software name="burger">
-		<description>Burger Shop</description>
+	<software name="cannonbl">
+		<description>Cannon Ball</description>
 		<year>1983</year>
 		<publisher>Bandai</publisher>
-		<info name="usage" value="Z and X to start and move; Space to pick up and put down" />
+		<info name="serial" value="0201028"/>
+		<info name="alt_title" value="キャノンボール"/>
+		<info name="usage" value="Press A for single player or D for two players. A or D to move, S to Shoot" /> <!-- Keyboard controls are glitchy and don't respond correctly, gamepad works fine. -->
```

Comment:
Legit sounds a `supported="partial"` +comment should be in notes field.

Reply:
I agree. I did some additional testing by running the game with -nojoy, and the game still had the same input problems. I'll amend this in a bit.